### DISCUSSION TITLE
Add class to MenuItemInterface

### DISCUSSION THREAD

**Discussion by creiner** [kevinpapst/TablerBundle#179] [Ideas]
Hey,
just an idea for more flexibility in the menu customization.

What do you think about a getClass() method in the Interface (+ default implementation) and the corresponding extension in the menu.html.twig (the content is added to the li class)?
(Maybe we name it getLayoutClass?)

What  do you think about this? 
If you are happy about it, I can provide an PR.

Best Regards
Christian

**Comment by kevinpapst** [comment]
Not exactly happy about adding breaking BC changes.
But we are likely introducing new interface methods in #176, so a new major release is required anyway.
What is your exact use-case?

**Comment by creiner** [comment]
I'm using it for adding classes to hide menu options on smaller display sizes. 
But in my opionen there are more use cases like adding badges, styling etc.
